docs/guides/migration-guide.md

```diff
@@ -8,128 +8,112 @@ search:
   boost: 2
 ---
 
-# Migrating from Prefect 1 to Prefect 2
+# Migrating from Prefect 1 to Prefect 3
 
-This guide is designed to help you migrate your workflows from Prefect 1 to Prefect 2. 
+Learn how to migrate your workflows from Prefect 1 to Prefect 3. Check out the [quickstart](https://docs.prefect.io/3.0/get-started/quickstart) for a high level overview.
+
+!!! Note "What about Prefect 2?"
+    Prefect 2 refers to the 2.x lineage of the open source prefect package, and Prefect 3 refers exclusively to the 3.x lineage of the prefect package. Neither version is strictly tied to any aspect of Prefect’s commercial product, [Prefect Cloud](https://docs.prefect.io/3.0/manage/cloud/index).
```

Comment:
perhaps we could add some note about how the jump from 2 to 3 is not nearly as big as 1 to 2

relatedly, I like what (I think) this is going for, which is "cloud works for both 2.x and 3.x, dont worry" but the way this sentence is worded is sort of confusing to me

Reply:
How about just

> Prefect Cloud is compatible with all 2.x and 3.x package versions.

Reply:
How about:
> Prefect Cloud is compatible with all 2.x and 3.x package versions. We recommend adopting Prefect 3.0 as it is significantly more performant than Prefect 2.x and includes new features.

By the way, I ripped the original wording directly from: https://docs.prefect.io/3.0/resources/whats-new-prefect-3

Reply:
Sounds good to me!